doc/source/User_guide/modeler.rst

```diff
@@ -1,26 +1,35 @@
 Modeler
 =======
 The AEDT 3D and 2D Modelers use object-oriented programming to create and manage objects. 
-You can use getters and setters to create an object and change its properties:
+You can use getters and setters to create an object and change its properties.
 
-.. code:: python
+In the following example we create a box and change the color to red:
```

Comment:
```suggestion
The following code creates a box and changes the color to red:
```
Don't use "we" but "you" or change the subject to something else that can use an action verb--as I did in my suggestion.